test/validate-test-promise.js

```diff
@@ -0,0 +1,74 @@
+  var assert, error_fixture, error_refract, fs, path, protagonist, valid_fixture, warning_fixture, warning_refract;
```

Comment:
Intentation seems wrong here, this is indented by a space?

Should change the import block to be:

```js
const fs = require('fs');
const path = require('path');
const { assert } = require('chai');
const protagonist = require('./protagonist');

const valid_fixture = path.join(__dirname, './fixtures/sample-api.apib');
...
```

So:

- Use `const` instead of var as these do not change and are constant.
- Remove extraneous newline between require statements.
- Remove the leading indentation space.
- Add separation between the requires and path join constants

Reply:
One other thing, which is probably why windows tests are failing, you are using path.join but also hard coding the path separator:

`path.join(__dirname, './fixtures/sample-api.apib')`

should become:

`path.join(__dirname, 'fixtures', 'sample-api.apib')`